indico/modules/events/editing/client/js/management/editable_type/EditableList.module.scss

```diff
@@ -0,0 +1,82 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@import 'base/palette';
+
+$border: 1px solid $pastel-gray;
+
+.editable-list {
+    .table {
+        font-family: 'Muli', 'Helvetica Neue', Arial, Helvetica, sans-serif;
+        margin-top: 20px;
+        margin-bottom: 50px;
+    }
+
+    :global(.ReactVirtualized__Table__Grid) {
+        outline: none;
+    }
+
+    :global(.ReactVirtualized__Table__headerRow) {
+        font-weight: 700;
+        text-transform: uppercase;
+        display: flex;
+        border-top: $border;
+        border-bottom: $border;
+    }
+
+    :global(.ReactVirtualized__Table__row) {
+        display: flex;
+        border-bottom: $border;
+    }
+
+    :global(.ReactVirtualized__Table__headerTruncatedText) {
+        display: inline-block;
+        max-width: 100%;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        overflow: hidden;
+    }
+
+    :global(.ReactVirtualized__Table__headerColumn),
+    :global(.ReactVirtualized__Table__rowColumn) {
+        margin-right: 10px;
+        align-items: center;
+        display: flex;
+    }
+
+    :global(#status.ReactVirtualized__Table__headerColumn) {
+        justify-content: center;
+    }
+
+    :global(.rowColumn_tooltip) {
```

Comment:
If it's a function that is not part of React Virtualized then it could be named the same way we use for other css classes. For example: `row-column-tooltip` or something like this.